recipes/default.rb

```diff
@@ -35,27 +35,34 @@
   notifies :reconfigure, 'chef_server_ingredient[supermarket]'
 end
 
-if node['supermarket_package']['package_source']
-  pkgname = ::File.basename(node['supermarket_package']['package_source'])
+if node['supermarket_omnibus']['package_url']
```

Comment:
yes @irvingpop and it's not a good enough reason to break everyone :) i was cargo-lifting from the chef-ingredient rework i did for the push-jobs cookbook